pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -581,3 +569,94 @@ func describeServicePorts(spec kapi.ServiceSpec) string {
 		return " ports " + strings.Join(pairs, ", ")
 	}
 }
+
+func loadServices(g osgraph.Graph, graphLock sync.Mutex, namespace string, kclient kclient.Interface, client client.Interface) error {
```

Comment:
The code falls out a lot simpler like this.  The alternatives are:
 1.  Load the resources into some intermediate, typed structure and then have separate logic to add them to the graph from there.
 1.  Return a generic runtime.Object, then have some sort of global registration for the graph that allows adding generic runtime.Objects (or interface{}).  I've done that for sane testing, but you've been against centralizing control like that before so I kept it out of the main code path.